Features/exacte_match.feature

```diff
@@ -46,7 +50,7 @@ Abstract scenario: geen exacte match ondanks exacteMatch=true
   Dan bevat het resultaat geen objecten
 
 Abstract Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor
-  Gegeven een straat met postcode 1234AA en twee objecten met huisnummer 3
+  Gegeven een object met postcode 1234AA en twee objecten met huisnummer 3
```

Comment:
Hier moeten we bij vermelden, dat één of beide objecten dan een huisletter en/of huisnummertoevoeging moet hebben, waardoor de nummeraanduiding uniek is. 
Dat er ook scenario's kunnen zijn waarbij bv. een huisletter gelijk is en de huisnummertoevoeging het verschil maakt, is dan niet meer zo relevant.

Reply:
dit is juist een voorbeeld om aan te geven wat er gebeurt als er echt 2 identieke huisnummers bestaan. Dat komt in enkele gevallen voor.

Reply:
ok